packages/color-wheel/README.md

```diff
@@ -1,64 +1,283 @@
 [![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/color-wheel?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/color-wheel)
 [![Try it on Stackblitz](https://img.shields.io/badge/Try%20it%20on-Stackblitz-blue?style=for-the-badge)](https://stackblitz.com/edit/vitejs-vite-cqyqwmbq)
 
-```
+```bash
 yarn add @spectrum-web-components/color-wheel
 ```
 
 Import the side effectful registration of `<sp-color-wheel>` via:
 
-```
+```javascript
 import '@spectrum-web-components/color-wheel/sp-color-wheel.js';
 ```
 
 When looking to leverage the `ColorWheel` base class as a type and/or for extension purposes, do so via:
 
-```
+```javascript
 import { ColorWheel } from '@spectrum-web-components/color-wheel';
 ```
 
-## Color Formatting
+### Anatomy
 
-When using the color elements, use `el.color` to access the `color` property, which should manage itself in the color format supplied. If you supply a color in `rgb()` format, `el.color` should return the color in `rgb()` format, as well.
+The color wheel consists of several key parts:
 
-The current color formats supported are as follows:
+- A circular track displaying the full spectrum of hues
+- A draggable handle that indicates the current hue selection
+- Accessible labels for screen readers
+- Optional custom gradient slot for advanced styling
 
-- Hex3, Hex4, Hex6, Hex8
-- HSV, HSVA
-- HSL, HSLA
-- RGB, RGBA
-- Strings (eg "red", "blue")
+```html
+<sp-color-wheel></sp-color-wheel>
+```
 
-**Please note for the following formats: HSV, HSVA, HSL, HSLA**
-When using the HSL or HSV formats, and a color's value (in HSV) is set to 0, or its luminosity (in HSL) is set to 0 or 1, the hue and saturation values may not be preserved by the element's `color` property. This is detailed in the [colorjs documentation](https://colorjs.io/docs/). Seperately, the element's `value` property is directly managed by the hue as represented in the interface.
+#### Custom Gradient
 
-## Example
+You can provide a custom gradient to replace the default color wheel appearance using the `gradient` slot:
 
 ```html
-<sp-color-wheel></sp-color-wheel>
+<sp-color-wheel>
+    <div slot="gradient" class="custom-gradient">
+        <!-- Custom SVG or gradient implementation -->
+    </div>
+</sp-color-wheel>
 ```
 
-### Disabled
+### Options
 
-A color wheel in a disabled state shows that an input exists, but is not available in that circumstance. This can be used to maintain layout continuity and communicate that the wheel may become available later.
+#### Properties
+
+The color wheel supports several properties for configuration:
+
+##### Value (Hue)
+
+Control the hue value directly (0-360 degrees):
 
 ```html
-<sp-color-wheel disabled></sp-color-wheel>
+<sp-color-wheel value="180"></sp-color-wheel>
+```
+
+##### Color Values
```

Comment:
not readable at all. We should not go beyond h3

Reply:
@Rajdeepc you are mistaken.

Per the documentation guidelines and structure which were written in consultation with accessibility and design. We avoid heading 5 in favor of tabs when possible, but heading 3 and heading 4 are absolutely necessary for semantics and consistent structure.

- https://opensource.adobe.com/spectrum-web-components/guides/adding-component/#documentation-standards
- https://opensource.adobe.com/spectrum-web-components/guides/adding-component/#documentation-structure